request.go

```diff
@@ -247,3 +247,25 @@ type RefundReq struct {
 	Amount    int64  `json:"amount"`
 	Reason    string `json:"reason"`
 }
+
+// Subscribe : Represent Subscribe object payload (request and response)
+type Subscribe struct {
+	ID          string      `json:"id,omitempty"`
+	Name        string      `json:"name"`
+	Amount      string      `json:"amount"`
+	Currency    string      `json:"currency"`
+	CreatedAt   string      `json:"created_at,omitempty"`
+	Status      string      `json:"status,omitempty"`
+	Token       string      `json:"token"`
+	PaymentType PaymentType `json:"payment_type"`
+	Schedule    Schedule    `json:"schedule"`
+}
+
+// Schedule : Represent Schedule object payload
+type Schedule struct {
```

Comment:
Can we name it `ScheduleDetail` to make it more consistent with the rest of struct?